index.html

```diff
@@ -33,6 +33,62 @@
       </nav>
   </header>
   <main>
```

Comment:
:red_circle: After the closing tag` </main>` there is a ` <script>` tag  used to embed JavaScript code. Currently, if you try to run your code in your project in the browser there is a greater chance for your script to not work which was my case before I fix the bug myself in order to review the project. Please make sure you always add the dot to the src attribute when you are specifying a relative path. This will help to prevent errors in the future. 

The dot (.) is important because it tells the browser to look for the file relative to the current file. Without the dot, the browser would look for the file in the root directory of the website, which is not where the file actually is.

I have updated  that line with the below code 

` <script src="./assets/js/script.js" type="application/javascript"></script>`

**Notice the dot before the` /assets`**

The dot made the difference in this case because the file script.js is located in the assets/js directory, which is one level down from the current file. As you had not added the dot, the browser was looking for the file in the root directory, which caused an error.